app/ui/browser/views/navbar/location.jsx

```diff
@@ -169,12 +171,22 @@ class Location extends Component {
     this.setState({ focusedURLBar: false });
   }
 
-  handleURLBarKeyDown(ev) {
+  handleURLBarKeyDown(maxCompletions, ev) {
     if (ev.keyCode === 13) { // enter
       this.props.navigateTo(fixURL(ev.target.value));
     } else if (ev.keyCode === 27) { // esc
       this.props.onLocationReset();
       ev.target.select();
+    } else if (ev.keyCode === 40 && maxCompletions > 0) { // down
```

Comment:
Let's use constants for these keycodes, we use them in a few places (can follow up)